rspace/src/test/scala/coop/rchain/rspace/ReplayRSpaceTests.scala

```diff
@@ -663,57 +649,14 @@ trait ReplayRSpaceTests extends ReplayRSpaceTestsBase[String, Pattern, String, S
       mm.get(cr).map(_.size).value shouldBe 2
 
       replaySpace.consume(channels, patterns, k, persist = false)
-
-      mm.get(cr).map(_.size).value shouldBe 1
-
       replaySpace.consume(channels, patterns, k, persist = false)
-
-      mm.get(cr) shouldBe None
-  }
-
-  "producing" should "correctly remove things from replay data" in withTestSpaces {
```

Comment:
What's the reason for removing this test?

Reply:
It is already covered by the test above.